impactcommon/exogenous_economy/gdppc.py

```diff
@@ -85,7 +85,8 @@ def get_iso_timeseries(self, iso):
             for year in range(baseline_year + 1, 2100 + 1):
                 yearindex = int((year - 1 - baseline_year) / 5) # Last year's growth
                 growthrate = df_growth.loc[df_growth.yearindex == yearindex].growth.values
-                gdppcs.append(gdppcs[-1] * growthrate)
+                new_gdppcs = gdppcs[-1] * growthrate
```

Comment:
This will always be a single value, so either call it `new_gdppc` or combine these two lines as a single line (that would be my preference, but it's just aesthetics).

Reply:
Renamed this variable in 73d69ed as suggested. Marking as resolved.